stages/org.osbuild.wpa_supplicant

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Write wpa_supplicant configuration file.
+
+Allows a person to write SSIDs to the `wpa_supplicant` configuration file which
+lets an image connect to wifi on first boot.
+
+WARNING: This stage expects the passphrase and SSID 'in-the-clear' in your manifest.
```

Comment:
This is also related to https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/106 as well as https://systemd.io/CREDENTIALS/